pytorch_amazon_network_analysis/public/json_converter.py

```diff
@@ -0,0 +1,58 @@
+"""
+ Purpose of this file is to create product_reviews.json file which we use to create cypher queries.
+ If you want to use some test some other cypher queries, different from ones we prepared, go to the
+ http://snap.stanford.edu/data/amazon/, download, unzip and save your dataset in `data` folder. Afterwards,
+ you can process it with json_converter.py
```

Comment:
This is a bit unclear, why do we need a different dataset to use different queries?

Reply:
Because not all datasets are the same, you can choose from shoe reviews to tech side reviews.